test/wdio.conf.js

```diff
@@ -0,0 +1,114 @@
+/* jshint node: true */
+
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var del = require('del');
+var Q = require('q');
+var request = require('request');
+var selenium = require('selenium-standalone');
+var screenshotPath = path.join('test', 'errorshots');
+
+
+/**
+ * Start a selenium server if it doesn't exist
+ *
+ * @returns {promise} A deferred promise
+ */
+function startSelenium () {
+  var deferred = Q.defer();
+  var statusUrl = 'http://localhost:4444/wd/hub/status';
+
+  request(statusUrl, function (error) {
+    // selenium not started, start it
+    if (error) {
+      selenium.install({}, function (err) {
+        if (err) { deferred.reject(err); }
+
+        selenium.start(function (err, child) {
+          if (err) { deferred.reject(err); }
+
+          selenium.child = child;
+          deferred.resolve();
+        });
+      });
+    }
+
+    // already started, return promise
+    deferred.resolve();
+  });
+
+  return deferred.promise;
+}
+
+
+var config = {
+
+  // ==========
+  // Test Files
+  // ==========
+  specs: [
+    './test/features/**/*.feature'
+  ],
+
+  // ============
+  // Capabilities
+  // ============
+  capabilities: [{
+    browserName: 'chrome'
+  }, {
+    browserName: 'firefox'
+  }, {
+    browserName: 'phantomjs'
+  }],
+
+  // ===================
+  // Test Configurations
+  // ===================
+  logLevel: 'silent',
```

Comment:
Not sure we do at this stage. This is the level of output that is displayed for the selenium interactions and are actually quite noisy around the basic test logging.

See [logLevel](http://webdriver.io/guide/getstarted/configuration.html#logLevel)